### PR TITLE
Correct meeting times

### PR DESCRIPTION
**Pull request by puerco**
This PR corrects the meeting time for the OpenVEX sig and adds some links to a time converter and the openssf calendar. It also fixes some funkiness in the markdown headlines.

/cc @SecurityCRob 

Signed-off-by: Adolfo García Veytia (Puerco) <puerco@chainguard.dev>

### CODE REVIEW

**Comment by jspeed-meyers** [comment]
@puerco, are you looking for a reviewer? I'm glad to be one. I can also find "volunteers" :)

**Comment by puerco** [comment]
Yes @jspeed-meyers thank you, just a second pair of eyes before merging it would be helpful :pray:

**Review by jspeed-meyers — APPROVED** [review]
LGTM!

**Comment by jspeed-meyers** [comment]
@SecurityCRob, I've run into you because of an Atlantic Council meeting recently, is that right? Howdy to you on the internet!

Would you mind merging this PR when you get a hot sec? I thought my review was good enough to get @puerco the opportunity to merge this. Apparently, not :(

**Pull request merged**
